google/cloud/storage/async/writer.cc

```diff
@@ -65,18 +65,19 @@ future<StatusOr<AsyncToken>> AsyncWriter::Write(AsyncToken token,
       });
 }
 
-future<StatusOr<storage::ObjectMetadata>> AsyncWriter::Finalize(
+future<StatusOr<google::storage::v2::Object>> AsyncWriter::Finalize(
     AsyncToken token, WritePayload payload) {
-  if (!impl_) return StreamError<storage::ObjectMetadata>(GCP_ERROR_INFO());
+  if (!impl_) return StreamError<google::storage::v2::Object>(GCP_ERROR_INFO());
   auto t = storage_internal::MakeAsyncToken(impl_.get());
-  if (token != t) return TokenError<storage::ObjectMetadata>(GCP_ERROR_INFO());
+  if (token != t)
+    return TokenError<google::storage::v2::Object>(GCP_ERROR_INFO());
```

Comment:
Nit: Braces now that the line has been broken?

Reply:
Done.